pkg/metrics/collector.go

```diff
@@ -47,6 +48,35 @@ var (
 			return vs.UnknownCount
 		},
 	}
+	exposedSecretLabels = []string{
+		"namespace",
+		"name",
+		"image_registry",
+		"image_repository",
+		"image_tag",
+		"image_digest",
+		"severity",
+	}
+	exposedSecretDesc = prometheus.NewDesc(
+		prometheus.BuildFQName("trivy", "image", "exposedsecrets"),
```

Comment:
@chen-keinan Not sure about this. Exposed secrets will exist per image and also per resource. The way I see it we will have two different metric entries for it, like we would have for the resources themselves. No?

```
trivy", "image", "exposedsecrets -> for secrets inside images
trivy", "resource", "exposedsecrets -> for secrets inside resources
```

because although the same report, each report will be tied to a different resource, one for image, and another one for a k8s resoure.

Reply:
Ok , then need probebly to think on a different naming for exposed data on resource , to avoid confusion